ToDoAppWithCoreData/Controllers/ListViewController.swift

```diff
@@ -0,0 +1,178 @@
+
+import UIKit
+import CoreData
+
+protocol ListViewControllerDelegate {
```

Comment:
Delegateは循環参照を防ぐためにweakで保持する必要があるので、class protocolにした方が良いです。
```suggestion
protocol ListViewControllerDelegate: class {
```